build_tools/packaging/linux/package.json

```diff
@@ -207,103 +256,2443 @@
+    "Group": "unknown",
+    "License": "MIT",
+    "Vendor": "Advanced Micro Devices, Inc",
+    "Homepage": "https://github.com/ROCm/rocm-systems",
+    "Components": [
+      "lib",
+      "run",
+      "dev",
+      "doc"
+    ],
+    "Includes": [
+      "rocm-core",
+      "rocm-smi-lib",
+      "rocprofiler-register",
+      "half"
+    ],
+    "Artifact": "base",
+    "Gfxarch": "False",
+    "Composite": "yes"
+  },
+  {
+    "Package": "amd-llvm",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [
+      "python3",
+      "libc6"
+    ],
+    "RPMRequires": [
+      "python3",
+      "glibc"
+    ],
+    "Maintainer": "ROCm Dev Support <rocm-dev.support@amd.com>",
+    "Description_Short": "ROCm llvm compiler",
+    "Description_Long": "ROCm llvm compiler with rocm-llvm, comgr and hipcc",
+    "Section": "devel",
+    "Priority": "optional",
+    "Group": "unknown",
+    "License": "ASL 2.0 with exceptions",
+    "Vendor": "Advanced Micro Devices, Inc",
+    "Homepage": "https://github.com/ROCm/llvm-project",
+    "Components": [
+      "lib",
+      "run",
+      "doc"
+    ],
+
+    "Includes": [
+      "rocm-llvm",
+      "comgr",
+      "hipcc"
+    ],
+    "Artifact": "amd-llvm",
+    "Gfxarch": "False",
+    "Disable_DWZ": "True",
+    "Composite": "Yes"
+  },
+  {
+    "Package": "amd-llvm-devel",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [
+      "python3",
+      "libc6",
+      "amd-llvm"
+    ],
+    "RPMRequires": [
+      "python3",
+      "glibc",
+      "amd-llvm"
+    ],
+    "Maintainer": "ROCm Dev Support <rocm-dev.support@amd.com>",
+    "Description_Short": "Amd llvm compiler libraries",
+    "Description_Long": "Contains rocm-llvm, comgr and hipcc development files",
+    "Section": "devel",
+    "Priority": "optional",
+    "Group": "unknown",
+    "License": "ASL 2.0 with exceptions",
+    "Vendor": "Advanced Micro Devices, Inc",
+    "Homepage": "https://github.com/ROCm/llvm-project",
+    "Components": [
+      "dev"
+    ],
+    "Includes": [
+      "rocm-llvm-devel"
+    ],
+    "Artifact": "amd-llvm",
+    "Gfxarch": "False",
+    "Composite": "Yes"
+  },
+  {
+    "Package": "rocm-hsa-runtime",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [
+      "libdrm-amdgpu-amdgpu1 | libdrm-amdgpu1",
+      "libnuma1",
+      "libelf1"
+    ],
+    "RPMRequires": [
+      "(libdrm-amdgpu or libdrm or libdrm_amdgpu1)",
+      "(libnuma1 or numactl-libs)"
+    ],
+    "Maintainer": "AMD HSA Support <dl.HSA-Runtime-Support@amd.com>",
+    "Description_Short": "AMD Heterogeneous System Architecture HSA - Linux HSA Runtime for Boltzmann (ROCm) platforms",
+    "Description_Long": "The HSA runtime is a thin, user-mode API that exposes the necessary interfaces to access and interact with graphics hardware driven by the AMDGPU driver set and the ROCK kernel driver.",
```

Comment:
What is "Boltzmann (ROCm)" here? Could that be replaced with just "ROCm"?

I see Boltzmann Initiative referenced at https://en.wikipedia.org/wiki/ROCm dating back to 2015... is the history still useful?

Reply:
Done. Removed Boltzmann and used ROCm